pkg/controller/replicaset/replica_set_test.go

```diff
@@ -1738,3 +1740,17 @@ func TestRemoveCondition(t *testing.T) {
 		}
 	}
 }
+
+func resourcePath(resource, namespace, name string) string {
+	path := fmt.Sprintf("/apis/%s/%s/", apps.SchemeGroupVersion.Group, apps.SchemeGroupVersion.Version)
```

Comment:
as is, this test breaks in environments where v1beta2 is not enabled (like in GKE prior to today)